documentation/src/main/asciidoc/topics/proc_configuring_global_persistent_location.adoc

```diff
@@ -0,0 +1,49 @@
+[id='configuring-global-persistent-location_{context}']
+= Configuring Global Persistent Location
+
+{brandname} preserves global state so that it can restore cluster topology and cached data after restart.
+If you enable global state in your {brandname} configuration, you should also configure a suitable location for {brandname} to persist global state.
+
+Global state is enabled by default with {brandname} Server, which persists cluster state to the `{server_home}/server/data` directory so it can perform graceful shutdowns.
+You do not need to configure a global persistent location for remote caches.
+
+For embedded caches, {brandname} defaults to the `user.dir` system property as the global persistent location, which is generally the directory where your application starts.
+You should always enable global state and configure a global persistent location for clustered embedded caches.
```

Comment:
Should we mention why? Is this due to conflicts in the user.dir ?

Reply:
@wburns I think it's kind of a best practice for preserving cluster state and choosing a sensible location as the default for embedded is probably more suitable for local caches. I'll try to reword this and make it clearer.